g11n-ws/vip-manager-i18n/src/main/java/com/vmware/vip/core/Interceptor/APIValidationInterceptor.java

```diff
@@ -55,8 +69,9 @@ public APIValidationInterceptor(Map<String, Object> allowedListMap) {
 	public boolean preHandle(HttpServletRequest request,
 			HttpServletResponse response, Object handler) throws Exception {
 		LOGGER.debug(request.getSession().getId());
-		String logOfUrl = "The request url is: " + request.getRequestURL();
-		String logOfQueryStr = "The request query string is: " + request.getQueryString();
+		String SingletonRequestID = getRequestId(request, this.clientRequestIds);
+		String logOfUrl = SingletonRequestID+"The request url is: " + request.getRequestURL();
```

Comment:
Suggest to merge these 2 logs into 1. Add thread id and request to this log, and remove thread id from line 99.